src/containers/shared/components/Transaction/NFTokenBurn/types.ts

```diff
@@ -0,0 +1,10 @@
+import { TransactionCommonFields } from '../types'
+
+export interface NFTokenBurn extends TransactionCommonFields {
+  NFTokenID: string
+  Owner?: string
+}
+
+export interface NFTokenBurnInstructions {
```

Comment:
I'm a bit confused why this is called `NFTokenBurnInstructions` - what is it instructions for? (Is there a simpler name?)

Reply:
"Instructions" is what the legacy code called the result of the parser functions.  I choose to keep it though open to better names